app/workers/webhook/destroy_worker.rb

```diff
@@ -1,6 +1,8 @@
 module Webhook
-  class DestroyJob < ApplicationJob
-    queue_as :webhook_destroy
+  class DestroyWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :low_priority, retry: 10
 
     def perform(user_id:, application_id:)
```

Comment:
Since we are moving this to Sidekiq we actually need to remove these symbols and instead use simple arguments. You can read more about this [in Sidekiq's Best Practices docs](https://github.com/mperham/sidekiq/wiki/Best-Practices#1-make-your-job-parameters-small-and-simple). Basically because everything is serialized we can't use symbols here.

Reply:
Ok, I will make the changes. But I have no idea about the reason for this test is failing. Do you have any clue?

Reply:
Yeah, we are using helpers for delayed job in that spec to run the background jobs for it. You need to convert them to using the new [sidekiq helpers](https://github.com/thepracticaldev/dev.to/blob/master/spec/support/sidekiq_test_helpers.rb) so that the new sidekiq worker is run instead.